<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>
 Copyright (C) 2015 Yannick Mueller

 This file is part of longscroll-qt.

 longscroll-qt is free software; you can redistribute it and/or modify it under
 the terms of the GNU Affero General Public License as published by the Free
 Software Foundation; either version 3, or (at your option) any later
 version.

 longscroll-qt is distributed in the hope that it will be useful, but WITHOUT ANY
 WARRANTY; without even the implied warranty of MERCHANTABILITY or
 FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero General Public License
 for more details.

 You should have received a copy of the GNU Affero General Public License
 along with longscroll-qt.  If not see http://www.gnu.org/licenses/.
 </comment>
 <class>BaseNavigatorWidget</class>
 <widget class="QFrame" name="BaseNavigatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="mainLayout">
   <item row="3" column="2">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>64</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="nextButton">
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="prevButton">
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>44</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="closeButton">
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>64</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>44</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" rowspan="4">
    <layout class="QHBoxLayout" name="infoLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
